bsip-0022.md

```diff
@@ -9,113 +9,439 @@
+  * Nobody should hold permanent votes from users who have irreversibly lost their keys.
+  * Non-voters are not included in the democratic process for government elections & past votes in previous government elections do not carry over to new government elections to avoid parties remaining permanently in power.
   * A gradual vote weight decay rather than an immediate removal would slow the loss of vote weight, providing a window of partial vote weight application for the witness/committee/proxy to utilise whilst they campaign for users to vote for them to stay in power. Likewise, the slow rate of vote weight reduction will reduce the odds of a low vote weight attack (malicious users voting malicious users into power).
-* Countering voter apathy
+* Potential counters to voter apathy
   * Make use of the BSIPs 19 and 20 (profit-sharing/dividends against MPA & UIA respectively) to only include users who have participated in the voting mechanism when distributing dividends to BTS holders (not applicable to MPA holders) or UIA holders (at the discretion of the UIA issuer performing the dividend payment).
-  * Fund the development of 'DPOSHub' style websites to improve the ease of witness/committee campaigning (encouraging competition/innovation) as well as discovery (enabling users to find new users they believe worthy of power within the BTS network).
+  * Fund the development of 'DPOSHub' style websites to improve the ease of witness/committee campaigning (encouraging competition/innovation) as well as enabling users to find users they believe worthy of holding roles of power on the BitShares platform).
+  * Making voting more prominent within the reference wallet UI (Github issues: [1934](https://github.com/bitshares/bitshares-ui/issues/1934) & [1932](https://github.com/bitshares/bitshares-ui/issues/1932)).
 
 ## Concerns of a lack of central communications channel
 
-* Some users have voiced concerns that because there isn't one mutually utilized central communications channel within the Bitshares community, we shouldn't implement vote weight decay.
-* I can sympathise with this concern, our community is distributed around the world & some countries block communication channels that others have access to (Russia rumoured to potentially block Telegram, China's "great firewall", venezuela blocking whatsapp, etc), likewise there are several language barriers between sub-communities.
-* To account for this concern, we should not implement this change without sufficient discussion nor consensus & transcript translation (and distribution of such information) for our users worldwide.
+* Some users have voiced concerns that because there isn't one mutually utilized central communications channel within the BitShares community, we shouldn't implement vote weight decay.
+* I can sympathise with this concern, our community is distributed around the world and there are several language barriers between sub-communities.
```

Comment:
Please re-word - IMO individual feelings should not be expressed here. The concerns (distribution + language barrier) should be mentioned of course.